tests/src/dir.targets

```diff
@@ -66,7 +66,6 @@
   build a run only for a higher priority testcase.
   -->
   <Target Name="Build" Condition="('$(CLRTestPriority)' &lt;= '$(CLRTestPriorityToBuild)') Or ('$(_CLRTestInServiceOfHigherPriority)')">
-    <MSBuild Projects="@(ProjectReference)" Properties="_CLRTestInServiceOfHigherPriority=true"/>
```

Comment:
What about the other parts of this targets file that use this property?  Also, is anything forcing the ProjectReference to build elsewhere?